src/utilities/util_compact.c

```diff
@@ -15,9 +15,11 @@
 static int
 usage(void)
 {
-    static const char *options[] = {"-?", "show this message", NULL, NULL};
+    static const char *options[] = {"-c config",
+      "a configuration string to be passed to WT_SESSION.compact", "-?", "show this message", NULL,
```

Comment:
Can you format the config options for better readability like
 "-c config", "a configuration ...",
 "-?", "Show this message",
 NULL, NULL

Reply:
@kommiharibabu, I cannot because this is reverted as soon as `s_all` runs.

Reply:
ok, thanks for the info.